src/paddle.cpp

```diff
@@ -2,8 +2,27 @@
 
 Paddle::Paddle() : Entity() {}
 
+Paddle::Paddle(PaddleControls pc) {
+	controls = pc;
+}
+
 Paddle::~Paddle() {}
 
 void Paddle::update(float frameTime) {
 	Entity::update(frameTime);
+
+	float yVelocity = 0.0f;
+
+	if (input->isKeyDown(controls.up)) {
+		yVelocity -= paddleNS::VELOCITY;
+	}
+
+	if (input->isKeyDown(controls.down)) {
+		yVelocity = paddleNS::VELOCITY;
+	}
```

Comment:
When moving up you set to -= velocity. But when moving down you set yvelocity = velocity. What are you trying to do here?

Are you trying to set the velocity to negative? `yVelocity = -paddleNS::VELOCITY`
Or did you forget to add a plus? `yVelocity += paddleNS::VELOCITY`